llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1135,7 +1135,14 @@ class VPPhiAccessors {
   const VPBasicBlock *getIncomingBlock(unsigned Idx) const;
 
   /// Returns the number of incoming values, also number of incoming blocks.
-  unsigned getNumIncoming() const { return getAsRecipe()->getNumOperands(); }
+  /// Note that at the moment, VPWidenIntOrFpInductionRecipes only have a single
```

Comment:
Looks like this still includes bits of https://github.com/llvm/llvm-project/pull/138472, which looks like it hasn't landed yet?

Reply:
#138472 simply adds additional iterators, but that is optional and not needed for the verification patch itself; they both need handling for additional recipes  in `getNumIncoming` though. Currently https://github.com/llvm/llvm-project/pull/138472 is more of a follow-up after this patch